### PR TITLE
Adding a new online resource

### PR DESCRIPTION
**Pull request by arthurakay**
I made a PR against this project, but the bot told me to do it here instead: https://github.com/OWASP/www-project-vulnerable-web-applications-directory/pull/18

### CODE REVIEW

File: src/data/online.json [hunk 1]
```diff
@@ -491,5 +491,21 @@
...
+	{
+		"url": "https://cyberscavengerhunt.com",
+		"name": "Cyber Scavenger Hunt",
+		"technology": [
+			"Javacript",
+			"React"
+		],
+		"references": [
+			{
+				"name": "GitHub",
```
Comment (kingthorin) [inline src/data/online.json:504]: Should be one of these enum values:
https://github.com/OWASP/OWASP-VWAD/blob/6628545a81071b32ce9da00d312298b4dc586c19/schema.json#L91-L102

File: src/data/online.json [hunk 2]
```diff
@@ -491,5 +491,22 @@
...
+				"url": "https://github.com/arthurakay/cyberscavengerhunt"
+			}
+		],
+		"author": "Arthur Kay",
+		"notes": "A simple scavenger hunt to learn about pentesting a website or web application.",
+		"badge": null
```
Comment (kingthorin) [inline src/data/online.json:510]:
```suggestion
		"badge": "arthurakay/cyberscavengerhunt"
```

File: src/data/online.json [hunk 3]
```diff
@@ -491,5 +491,22 @@
...
+		"technology": [
+			"Javacript",
+			"React"
+		],
+		"references": [
+			{
+				"name": "download",
+				"url": "https://github.com/arthurakay/cyberscavengerhunt"
+			}
+		],
```
Comment (kingthorin) [inline src/data/online.json:507]: Is there a live one?

If not and this is meant to be locally hosted you should actually add it to the offline list.

Comment (arthurakay) [reply src/data/online.json:507]: It's live at https://cyberscavengerhunt.com -- see line 496.

I wasn't entirely sure if I needed to add the GitHub code or not (you certainly can run it locally)

Comment (kingthorin) [reply src/data/online.json:507]: Doh totally missed that, thanks!

**Pull request merged**